build.gradle

```diff
@@ -207,7 +207,7 @@ def versions = [
 
 dependencies {
 
-    implementation 'com.github.hmcts:ecm-common:0.1.158'
```

Comment:
The latest version with log4j fix is 0.1.160

Reply:
incremented version